package.json

```diff
@@ -7,7 +7,8 @@
     "dev": "next",
     "build": "next build",
     "start": "next start -p $PORT",
-    "test": "jest"
+    "test": "jest",
+    "lint:fix": "eslint './src/**/*.{ts,tsx}'"
```

Comment:
Does this work? We don't have a src folder...

Reply:
The script name should also be changed to match the step in the circle CI config (or the step should be changed to match the script name). Best to change the script name to lint since I don't think it will fix lint errors...?

Reply:
changed to "lint" and removed './src/**/*  and replaced with .